src/components/estimated-call/DepartureTime.tsx

```diff
@@ -37,7 +39,9 @@ export const DepartureTime = ({departure}: DepartureTimeProps) => {
               ? 'body__primary--strike'
               : 'body__primary--bold'
           }
-          color={departure.cancellation ? 'secondary' : 'primary'}
+          color={
+            color ? color : departure.cancellation ? 'secondary' : 'primary'
+          }
```

Comment:
This means that if a color is sent in, then there is no difference in text for `cancellation` anymore? 🤔

Reply:
The colour is used in the header which is the only one sending it in, the header as is now remains grey no matter the theme, the colour passed in is the one that contrasts, but it applies to both texts including cancellation, during the test the grey colour did not fit well (cancellations) in light mode.

![image](https://github.com/user-attachments/assets/4be8456e-c7a2-4985-87fe-20ed6e771e90)

Reply:
A PR to fix this was made on https://github.com/AtB-AS/mittatb-app/pull/4867